hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -51,6 +51,7 @@
 
 __virtualname__ = 'nebula'
 __RESULT_LOG_OFFSET__ = {}
+__CUSTOM_MASK_KEY__ = 'ENV_VARS_TO_REDACT'
```

Comment:
Would it be possible to avoid hard coding it in the code?

Importing it somehow from profile would allow to change it more flexibly.